apps/omg_watcher/lib/challenger.ex

```diff
@@ -35,9 +34,11 @@ defmodule OMG.Watcher.Challenger do
   Returns challenge for an exit
   """
   @spec create_challenge(Utxo.Position.t()) :: {:ok, Challenge.t()} | {:error, :invalid_challenge_of_exit}
-  def create_challenge(utxo_exit) do
-    with {:ok, challenging_tx} <- DB.Transaction.get_transaction_challenging_utxo(utxo_exit) do
-      {:ok, Core.create_challenge(challenging_tx, utxo_exit)}
+  def create_challenge(Utxo.position(blknum, txindex, oindex) = utxo_exit) do
```

Comment:
you might want to remove `DB.Transaction.get_transaction_challenging_utxo` for completeness (and the stuff only that called/used)